include/openrave/units.h

```diff
@@ -80,7 +80,8 @@ OPENRAVE_API TimeUnit GetTimeUnitFromString(const std::string& pTimeUnit, TimeUn
 enum AngleUnit : int8_t
 {
     AU_Radian = 0,
-    AU_Degree = 1
+    AU_Degree = 1,
+    AU_Centidegree = 2,
```

Comment:
value should be `-2`
also need corresponding MR on schemaprofile repo to support this..

Reply:
Ok, I guess it's for comparing `AU_Degree > AU_Centidegree`